static/templates/settings/organization-settings-admin.handlebars

```diff
@@ -76,6 +76,14 @@
                   "prefix" "id_"
                   "is_checked" realm_send_welcome_emails
                   "label" admin_settings_label.realm_send_welcome_emails}}
+                {{ page_params.settings_send_digest_emails }}
```

Comment:
What's the story of this line -- it looks like it doesn't belong?

Reply:
Yes, sorry. That's a debug line that I left behind!